### DISCUSSION TITLE
Specifying adl vs adl64?

### DISCUSSION THREAD

**Discussion by waveofthought-code** [BowlerHatLLC/vscode-as3mxml#578] [Q&A] [answered]
I'm not sure how to target which debugger I want to use -- is it automatic based on the \<architecture\> xml tag?   Is it an extension setting somewhere I can change?  Is it an SDK thing outside of this extension's control?

**Comment by joshtynjala** [comment]
This is the first I've heard of adl64. With that in mind, the extension currently does not provide any kind of setting or other configuration option that might allow you to use it.

**Reply by waveofthought-code** [reply]
I just discovered it myself; I have no idea when it was introduced by Harman

**Comment by waveofthought-code** [comment, marked as answer]
Thread necro a year later because I found a solution awhile back and forgot to post it:

_in launch.json:_
```
{
  "type": "swf",
  "request": "launch",
  "name": "Run App With 64-bit ADL",
  "program": "your-application.xml",
  "runtimeExecutable": "path-to-airsdk/bin/adl64.exe",
  "args" : []
}
```
